core/templates/dev/head/components/common-layout-directives/common-elements/sharing-links.directive.ts

```diff
@@ -79,7 +78,7 @@ oppia.directive('sharingLinks', [
 
           ctrl.escapedTwitterText = (
             HtmlEscaperService.unescapedStrToEscapedStr(
-              ctrl.getTwitterText()));
+              constants.DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR));
```

Comment:
Use generated angular constant please. (just `DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR` that you need to import)

Reply:
Done